test/gtest/ucp/test_ucp_wakeup.cc

```diff
@@ -118,6 +118,10 @@ UCS_TEST_P(test_ucp_wakeup, tx_wait, "ZCOPY_THRESH=10000")
     void *sreq, *rreq;
 
     sender().connect(&receiver(), get_ep_params());
+    /* since the test waits on receiver's worker for events, it may happen
+     * that the test hangs due to the lack of the progress on receiver's
+     * worker (e.g. TCP requires the progress for CM events) */
+    flush();
```

Comment:
If this is TCP specific behavior, better to disable the test by capability check since TCP doesn't support ZCOPY

Reply:
It looks like `check_caps` can be applied only for UCT. I don't find a way to extract `UCT_IFACE_FLAG_AM_ZCOPY` capability from UCP code - i.e. looks like a UCP user doesn't know which capabilities are supported by the UCT layer.

Reply:
I see... Then need to skip the test by TL name until usync wireup is not implemented there. This test is useless with flush.

Reply:
sure, will disable.
btw, I found the issue with `iface::outstanding` counter, debugging this - probably it may solve the hang here.